src/ARMarker.js

```diff
@@ -50,12 +54,22 @@ class ARMarker extends HTMLElement {
         }else{
             this.video = document.createElement("video");
             this.video.src = this.contentProps.src;
+
             this.video.loop = true;
```

Comment:
With the previous changes here you can do:
```suggestion
            this.video.loop = this.contentProps.loop;
```